conf/rconfc.c

```diff
@@ -2734,14 +2744,6 @@ static void rconfc_read_lock_complete(struct m0_rm_incoming *in, int32_t rc)
 	rlx = rconfc->rc_rlock_ctx;
 	if (rc != 0) {
 		M0_LOG(M0_ERROR, "Read lock request failed with rc = %d", rc);
-		/*
-		 * RPC connection to RM may be lost, or there is no remote RM
-		 * service anymore to respond to read lock request, so need to
-		 * handle this to prevent read lock context from further
-		 * communication attempts
-		 */
-		M0_ASSERT(rlock_ctx_is_online(rlx));
-		rlock_ctx_creditor_unset(rlx);
 	}
```

Comment:
Question:
Do we still need this if block ? We are not doing anything now apart from error log. 
I see rc is also printed in entry log of this function.

Reply:
It was the same as previous implementation. Also, it's good to have a ERROR log for failure along with CALL log.